practice/case.py

```diff
@@ -0,0 +1,146 @@
+    >>> slice_words_apart('DO_NOT_HUG_ME_I_AM_SCARED')
+    'DO NOT HUG ME I AM SCARED'
+    """
+    if '-' in unsliced_phrase:
+        sliced_list = unsliced_phrase.split('-')
+    elif '_' in unsliced_phrase:
+        sliced_list = unsliced_phrase.split('_')
+    else:
+        sliced_list = re.findall('[A-Z][^A-Z]*', unsliced_phrase)
+    spaced_phrase = ' '.join(sliced_list)
+    return spaced_phrase
+
+
+def standardize_phrase_lower(phrase):
+    """Sets the phrase into lowercase letters for easy manipulation
+
+    >>> standardize_phrase_lower('You are the weakest link Goodbye')
+    'you are the weakest link goodbye'
+    """
+    lower_phrase = phrase.lower()
+    return lower_phrase
+
+
+def alter_capitalization_to_new_case(phrase, case):
+    """Checks for the desired case, then capitalizes appropriately.
+    >>> alter_capitalization_to_new_case('apple core', 'CONSTANT_CASE')
+    'APPLE CORE'
+    >>> alter_capitalization_to_new_case('want some more', 'CamelCase')
+    'Want Some More'
+    """
+    if case == 'CamelCase':
+        capitalized_phrase = phrase.title()
+    elif case == 'CONSTANT_CASE':
+        capitalized_phrase = phrase.upper()
+    else:
+        capitalized_phrase = phrase
+    return capitalized_phrase
+
+
+def paste_phrase(phrase, new_case):
+    """Removes spaces from phrase
+    >>> paste_phrase('what is your favorite meme','snake_case')
+    'what_is_your_favorite_meme'
+    >>> paste_phrase('WHO DARES WINS','CONSTANT_CASE')
+    'WHO_DARES_WINS'
+    >>> paste_phrase('do not taser me bro','kebab-case')
+    'do-not-taser-me-bro'
+    >>> paste_phrase('What The Rock Is Cooking','CamelCase')
+    'WhatTheRockIsCooking'
+    """
+    if new_case == 'CamelCase':
+        pasted_phrase = str(phrase).replace(' ', '')
+    elif new_case == 'CONSTANT_CASE' or new_case == 'snake_case':
+        pasted_phrase = str(phrase).replace(' ', '_')
+    elif new_case == 'kebab-case':
+        pasted_phrase = str(phrase).replace(' ', '-')
+    return pasted_phrase
+
+
+def output(old_phrase, new_phrase, old_case, new_case):
+    """Outputs the translation.
+    >>> output('i-am-a-camel', 'IAmACamel', 'kebab-case', 'CamelCase')
+    i-am-a-camel translated from kebab-case to CamelCase is IAmACamel
+    """
+    print('{}, translated from {} to {} is {}!'.format(old_phrase, old_case, new_case, new_phrase))
+
+
+def main():
+    """Main Function"""
+    phrase_to_translate = input_phrase_to_translate()
+    new_case = input_new_case()
+    old_case = check_old_case(phrase_to_translate)
+    phrase_with_spaces = slice_words_apart(phrase_to_translate)
+    standardized_phrase_lower = standardize_phrase_lower(phrase_with_spaces)
+    appropriately_capitalized_phrase = alter_capitalization_to_new_case(standardized_phrase_lower, new_case)
+    final_phrase = paste_phrase(appropriately_capitalized_phrase, new_case)
+    output(phrase_to_translate, final_phrase, old_case, new_case)
+
+if __name__ == '__main__':
+    main()
```

Comment:
Check your pull requests right after you make them to see if they have extraneous files in them.